runtime/cudaq/platform/default/rest/helpers/braket/BraketExecutor.cpp

```diff
@@ -240,14 +257,20 @@ BraketExecutor::execute(std::vector<KernelExecution> &codesToExecute) {
           }
           auto resultsJson = nlohmann::json::parse(
               s3Response.GetResultWithOwnership().GetBody());
+
           auto c = serverHelper->processResults(resultsJson, taskArn);
 
           for (auto &regName : c.register_names()) {
+            std::cout << "Register name: " << regName << std::endl;
```

Comment:
Q: Is this expected to be printed to console? I'm not sure if the user will see it with remote job sunbmission...

Reply:
leftover printing, removed!